src/components/RichText/RichText.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import { Editor } from '@tinymce/tinymce-react'
+
+import 'tinymce/tinymce'
+
+// Basic tinyMCE theme & skins required for editor to display
+import 'tinymce/themes/silver/theme.min'
+import 'tinymce/skins/ui/oxide/skin.min.css'
+import 'tinymce/skins/ui/oxide/content.min.css'
+
+// Import required plugins
+import 'tinymce/plugins/autolink/plugin.min'
+import 'tinymce/plugins/lists/plugin.min'
+import 'tinymce/plugins/link/plugin.min'
+import 'tinymce/plugins/table/plugin.min'
+import 'tinymce/plugins/paste/plugin.min'
+import 'tinymce/plugins/charmap/plugin.min'
+
+interface Props {
+  /**
+   * id of the rich text editor component
+   * @default ""
+   */
+  id?: string
+  /**
+   *  Initial value of the rich text editor
+   * @default ""
+   */
+  value?: string
+  /**
+   * Defines whether the rich text editor should be enabled/disabled (default = false)
+   * @default ""
```

Comment:
I think that this should be `@default false`

Reply:
You're absolutely right :) Fixed in my latest commit.